src/contribute/release-guide.md

```diff
@@ -210,6 +214,19 @@ Check out the release branch.
 
 The rest of this guide assumes that commands are run in the root of a repository on `${BRANCH_NAME}` with the above environment variables set.
 
+### Update the Python SDK version
+
+In the master branch update Python SDK [version](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/version.py) to the next development version (e.g. `1.2.3.dev` to `1.3.0.dev`).
```

Comment:
In the master branch, update the Python SDK version identifier to the next development version.